pkg/base/replicated.go

```diff
@@ -228,6 +326,10 @@ func renderReplicatedHelmChart(kotsHelmChart *kotsv1beta1.HelmChart, upstreamFil
 		return nil, errors.Wrap(err, "failed to render local values for chart")
 	}
 
+	// Update the kots helmchart to only contain the rendered values.
+	kotsHelmChart.Spec.Values = mergedValues
```

Comment:
why is this needed? doesn't this change existing behavior?

Reply:
nevermind, I forgot about the comment to move the helm rendering into the `renderKotsKinds` function, this won't apply anymore after I finish that